models/maker/ethereum/maker_ethereum_sources.yml

```diff
@@ -27,4 +27,20 @@ sources:
       - name: vow_call_fess
         identifier: VOW_call_fess
       - name: vow_call_file
-        identifier: VOW_call_file
\ No newline at end of file
+        identifier: VOW_call_file
+      - name: dssvesttransferrable_call_create
+        identifier: DssVestTransferrable_call_create
+      - name: dssvesttransferrable_call_yank
+        identifier: DssVestTransferrable_call_yank
+      - name: dssvesttransferrable_evt_vest
+        identifier: DssVestTransferrable_evt_Vest
+      - name: mkr_evt_transfer
+        identifier: mkr_evt_Transfer
+  - name: prices
```

Comment:
this is the error here. prices as a source is already established in another source file, located [here](https://github.com/duneanalytics/spellbook/blob/main/models/prices/prices_sources.yml) -- no need to readd 👍

Reply:
in case you haven't used our CI testing before, you can see that [here](https://github.com/duneanalytics/spellbook/pull/2567/checks#step:7:11)

Reply:
i'd also highly recommend you run `dbt compile` on your local machine after updating models there, prior to pushing here, to ensure it completes successfully. even when compile completes, you can take the output code in target directory, which is plain sql, and run on dune to ensure it's as you'd like to see it